packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.scss

```diff
@@ -48,6 +48,13 @@
   .pf-chatbot__history-actions {
     transform: rotate(90deg);
   }
+
+  .pf-chatbot__menu-item--active {
+    background-color: var(--pf-t--global--background--color--action--plain--clicked);
+  }
```

Comment:
lets give the clicked state a border radius of 6px

Reply:
Also want 

```
.pf-chatbot__history.pf-v6-c-drawer .pf-v6-c-drawer__close .pf-v6-c-button {
    border-radius: 6px; // as a token 
}
```

Reply:
And then fix demo to track state of clicked item and update active state.